### DISCUSSION TITLE
Documentation doesn't mention the "subscription" fee

### DISCUSSION THREAD

**Discussion by giant69** [dvershinin/fds#16] [General]
When I try to install the fds package, I was asked to subscribe with a specific URL which leads to a paypal fee of 9.99$?

Please explain.

**Comment by dvershinin** [comment]
You can ignore it. Subscription is only required for binary packages from that repository. Which fds isn’t. Will update docs later on.

**Reply by giant69** [reply]
Hmm, I can't because the procedure didn't proceed since it checks a subscription of the IP-address of the system I'd like to install fds...

**Reply by dvershinin** [reply]
Can you please paste complete output from `yum install fds`

**Reply by linde002** [reply]
Here is the output I get on my machine.

  > Running transaction check
---> Package fds.noarch 0:0.0.20-1.el7 will be installed
--> Processing Dependency: python2-fds = 0.0.20-1.el7 for package: fds-0.0.20-1.el7.noarch
--> Processing Dependency: conntrack-tools for package: fds-0.0.20-1.el7.noarch
--> Processing Dependency: python2-CacheControl for package: fds-0.0.20-1.el7.noarch
--> Processing Dependency: python2-psutil for package: fds-0.0.20-1.el7.noarch
--> Running transaction check
---> Package conntrack-tools.x86_64 0:1.4.4-7.el7 will be installed
--> Processing Dependency: libnetfilter_cttimeout.so.1(LIBNETFILTER_CTTIMEOUT_1.1)(64bit) for package: conntrack-tools-1.4.4-7.el7.x86_64
--> Processing Dependency: libnetfilter_cttimeout.so.1(LIBNETFILTER_CTTIMEOUT_1.0)(64bit) for package: conntrack-tools-1.4.4-7.el7.x86_64
--> Processing Dependency: libnetfilter_cthelper.so.0(LIBNETFILTER_CTHELPER_1.0)(64bit) for package: conntrack-tools-1.4.4-7.el7.x86_64
--> Processing Dependency: libnetfilter_queue.so.1()(64bit) for package: conntrack-tools-1.4.4-7.el7.x86_64
--> Processing Dependency: libnetfilter_cttimeout.so.1()(64bit) for package: conntrack-tools-1.4.4-7.el7.x86_64
--> Processing Dependency: libnetfilter_cthelper.so.0()(64bit) for package: conntrack-tools-1.4.4-7.el7.x86_64
---> Package python2-CacheControl.noarch 0:0.12.6-1.el7 will be installed
--> Processing Dependency: python-lockfile for package: python2-CacheControl-0.12.6-1.el7.noarch
--> Processing Dependency: python2-msgpack for package: python2-CacheControl-0.12.6-1.el7.noarch
---> Package python2-fds.noarch 0:0.0.20-1.el7 will be installed
--> Processing Dependency: python2-cloudflare >= 2.7.1 for package: python2-fds-0.0.20-1.el7.noarch
--> Processing Dependency: python-netaddr for package: python2-fds-0.0.20-1.el7.noarch
--> Processing Dependency: python2-tqdm for package: python2-fds-0.0.20-1.el7.noarch
---> Package python2-psutil.x86_64 0:5.6.7-1.el7 will be installed
--> Running transaction check
---> Package libnetfilter_cthelper.x86_64 0:1.0.0-11.el7 will be installed
---> Package libnetfilter_cttimeout.x86_64 0:1.0.0-7.el7 will be installed
---> Package libnetfilter_queue.x86_64 0:1.0.2-2.el7_2 will be installed
---> Package python-netaddr.noarch 0:0.7.5-9.el7 will be installed
---> Package python2-cloudflare.noarch 0:2.7.1-1.el7 will be installed
--> Processing Dependency: PyYAML for package: python2-cloudflare-2.7.1-1.el7.noarch
---> Package python2-lockfile.noarch 1:0.11.0-17.el7 will be installed
---> Package python2-msgpack.x86_64 0:0.6.1-2.el7 will be installed
---> Package python2-tqdm.noarch 0:4.50.2-1.el7 will be installed
--> Running transaction check
---> Package PyYAML.x86_64 0:3.10-11.el7 will be installed
--> Processing Dependency: libyaml-0.so.2()(64bit) for package: PyYAML-3.10-11.el7.x86_64
--> Running transaction check
---> Package libyaml.x86_64 0:0.1.4-11.el7_0 will be installed
--> Finished Dependency Resolution
=======================================================================
Some packages in your transaction are brought to you by GetPageSpeed: 
 - python2-msgpack-0.6.1-2.el7.x86_64
To enable package installs, subscribe using the following link:

https://www.getpagespeed.com/repo-subscribe?server_ip=149.210.186.9
=======================================================================


Exiting because subscription is missing

**Reply by giant69** [reply]
Exactly! Thanks linde002.
Since I removed the repository, I could not reconstruct the output.

**Reply by dvershinin** [reply]
Thank you both for the details.

I've updated README for instructions on the free usage.
Only need to disable the `getpagespeed-extras` repository (this will keep the `getpagespeed-extras-noarch` enabled, this is where `fds` is). Should work:

    sudo yum -y install https://extras.getpagespeed.com/release-latest.rpm

Then *don't subscribe*.

Then, disable binary sub-repository:

    sudo yum -y install yum-utils
    sudo yum-config-manager --disable getpagespeed-extras

Install fds:

    sudo yum -y install fds

**Reply by linde002** [reply]
Works! Thanks for the quick response!
